scripts/bootstrap

```diff
@@ -23,8 +23,8 @@ echo "using rust version '$rust_version'"
 run curl --fail https://sh.rustup.rs -sSf | run sh -s -- -y --default-toolchain "none" --no-modify-path
 
 echo "looking for ic-cdk-optimizer"
-if ! command -v ic-cdk-optimizer
+if [[ ! "$(command -v ic-cdk-optimizer)" || "$(ic-cdk-optimizer --version)" == "ic-cdk-optimizer 0.3.4" ]]
 then
     echo "installing ic-cdk-optimizer"
-    run cargo install ic-cdk-optimizer --version 0.3.1
+    run cargo install ic-cdk-optimizer --version 0.3.4
```

Comment:
Here the message still says `looking for/installing ic-cdk-optimizer`, I think it'd be worth mentioning the version (otherwise the user may get confused as to why the script is re-installing the tool)

Reply:
Done.